app/controllers/tools_controller.rb

```diff
@@ -37,16 +36,8 @@ def set_body_id
     @body_id = 'tools'
   end
 
-  def tools
-    tool_class.order(published_at: :desc).live.published
-  end
-
-  def set_featured_tools
-    @featured_tools = tools.english.where.not(buy_url: nil).map { |tool| tool if tool.buy_url.present? }.compact
-  end
-
   def set_tools
-    @tools = tools.english.map { |tool| tool if tool.buy_url.blank? }.compact
+    @tools = tool_class.english.reorder(position: :desc).order(published_at: :desc).live.published.page(params[:page]).per(10)
```

Comment:
```suggestion
    @tools = tool_class.english.reorder(position: :asc).order(published_at: :desc).live.published.page(params[:page]).per(10)
```

you changed this back to `desc`, it needs to be `asc`.

also, can you move it to the tools concern as a scope similar to https://github.com/crimethinc/website/blob/90111fa77150f4abd7f00f54a8b68bd41d1a3296/app/models/book.rb#L5

Reply:
Gosh, I'm sorry I really understand the position as the higher the position, the higher is the priority !

Reply:
no worries :+1: